src/test/java/net/starschema/clouddb/jdbc/BQForwardOnlyResultSetFunctionTest.java

```diff
@@ -61,35 +60,46 @@
  */
 public class BQForwardOnlyResultSetFunctionTest extends CommonTestsForResultSets {
 
-  private static java.sql.Connection con = null;
+  private java.sql.Connection connection;
+  private java.sql.Connection standardSqlConnection;
   private java.sql.ResultSet resultForTest = null;
 
   Logger logger = LoggerFactory.getLogger(BQForwardOnlyResultSetFunctionTest.class);
   private Integer maxRows = null;
   private String defaultProjectId = null;
-  private BQConnection defaultConn = null;
+
+  private Connection connect(final String extraUrl) throws SQLException, IOException {
+    return ConnectionFromResources.connect("installedaccount1.properties", extraUrl);
+  }
 
   @Before
-  public void setup() throws SQLException, IOException {
-    Properties props =
-        BQSupportFuncts.readFromPropFile(
-            getClass().getResource("/installedaccount.properties").getFile());
-    String url = BQSupportFuncts.constructUrlFromPropertiesFile(props, true, null);
-    url += "&useLegacySql=false";
-    this.defaultProjectId = props.getProperty("projectid");
-    this.defaultConn = new BQConnection(url, new Properties());
+  public void setConnection() throws SQLException, IOException {
```

Comment:
I think having two @Before can be confusing since the order isn't guaranteed -- someone might cause dependency order.